frontend/src/apis/index.js

```diff
@@ -0,0 +1,39 @@
+// Is an abstraction to communicate with backend
+
+class API {
+    BASE_URL = '/api';
+    myFetch = (url, config) =>
+        window.fetch(this.BASE_URL + url, {
+            headers: { 'content-type': 'application/json' },
+            ...config,
+        });
+
+    async login({ pin, phoneNumber }) {
+        try {
+            return await this.myFetch(`/users/login`, {
+                method: 'POST',
+                body: JSON.stringify({
+                    pin,
+                    phoneNumber,
+                }),
+            }).then((res) => res.json());
```

Comment:
Our backend always returns JSON so the `res.json()` should be only in the fetch method.